TodoList/Extensions/UITableViewExtensions.swift

```diff
@@ -2,21 +2,30 @@ import UIKit
 
 extension UITableView {
     
+    /// Registers a nib object containing a cell with the table view. Use this function to avoid creating a reuse ID.
+    ///
+    /// - Parameters:- cellType: Name of xib to register
     func register<T: UITableViewCell>(cellType: T.Type) {
         let reuseIdentifier = String(describing: T.self)
         let nib = UINib(nibName: reuseIdentifier, bundle: Bundle(for: T.self))
         
         register(nib, forCellReuseIdentifier: reuseIdentifier)
     }
     
+    /// Returns a reusable table-view cell object and adds it to the table. Use this function to avoid creating a reuse ID.
```

Comment:
This is not accurate, it doesn't add a cell to the tableView, please read `dequeueReusableCell` documentation for better understanding of what it does.
Also I'd prefer to call it `UITableViewCell` instead of `table-view cell`